### PR TITLE
docs(design-system): 治理合同修订——管道与记账、已知边界、路线图勘误

### PR DESCRIPTION
**Pull request by kirozeng**
设计系统治理合同（#3609）的修订，**不占 DS 序号**；路线图见 `docs/design-rules/design-governance.md` §12。

### 摘要

把 `mivo-canvas-plugin` 仓 4/7 张 PR 的实战沉淀移植进 Cindy 治理合同（两个新节），并修掉三处编号/状态过期。零视觉、纯文档。

**为什么值得补这两节**：① 治理体系最常见的死法是后人把守卫红灯读成「设计被冻结了」，于是加豁免绕过去——§1.1 用「管道 vs 记账」把「哪些不许绕」和「哪些随时能改、只是要走合法路径」显式分开；② 不登记机器能力的边界，就会有人拿「机器没拦」当「合规」——§13 如实登记三条做不到的事。

### 变更类型

- [x] `docs` / `test` / `chore` 文档、测试或工程维护

### 范围

- 关联 Issue / 需求：设计系统治理（DS-1 = #3609 的后续修订）
- 本 PR 包含：
  - **新增 §1.1「管道与记账」**：管道（值只有一个出处、旧 Token ID 不删不改名、用户主题不改写磁盘、颜色走语义 token、台账只有一份）不许绕过；记账（具体色号、圆角三档当前成员、保护区观感）随时能改，合法路径 = 改数值真相源 → 同一 PR 更新守卫快照/台账/文档 → 按 §6 交证据 → 设计师批准。明确「守卫红灯不是禁令」，系统消灭的是**没人决定过的变化**
  - **新增 §13「已知边界」**：棘轮管不到内联样式与 canvas/xterm 一类自绘；「复用道路唯一」是纪律不是机器闸；语义分层是渐进的。开头写明登记目的 = 不得把「机器没拦」当合规证据
  - **勘误 1**：§2 旧编号「路线图 PR-8」→「DS-8」（命名规则定案后的遗留）
  - **勘误 2**：§12 依赖行的 DS-6 前置由**已关闭**的圆角裁决改为 §10「Permission 迁移余项」，并补「DS-7 棘轮基线受 §10 `radius` 主题覆盖裁决约束」——原文会让人误以为 DS-6 已解锁，而 §10 仍拦着
  - **勘误 3**：§12 路线图回填 DS-1 = ✅ #3609（2026-08-29 合入）
  - 索引表条目与版本记录同步
- 明确不包含：不新增/修改任何规则的**约束力**（§1.1 与 §13 是把既有实践显式化，不引入新门禁）；不改 `DESIGN.md`；不动 §2~§12 的章节编号（§1.1 走子节、§13 追加，既有引用不受影响）；不改任何代码、依赖、CI
- 用户可见变化：无
- 是否存在 breaking change：无

### UI 变化

不涉及（纯文档，产品界面零变化）。

- 引用的设计规范：不涉及：本 PR 只修订治理流程文档，未触碰 UI 代码路径；视觉规则仍以 `DESIGN.md` 为准，本 PR 未改动其正文。

### 自动验证

```text
pnpm test:unit:related
结果：PASS（435 pass / 0 fail / 1 skipped）

pnpm check:dco
结果：passed（1 commit signed off）
```

typecheck：未涉及任何 package 代码，按门禁规则自动跳过。

### 手工验证

- 逐条核对 §10 待裁决表与 §12 依赖行的一致性（本 PR 修的正是两者不同步）；
- 确认 §2~§12 编号未变，`design-decision-log.md` 与在办的 DS-2a 交接计划中的章节引用（§2.1 / §7 / §9 / §12）全部仍然有效；
- 两节新文字对照 `mivo-canvas-plugin` v4.1 方法论文档核实，未移植该仓特有机制（宿主 color-mix 公式、四格主题矩阵、forced 皮肤选择器——Cindy 是宿主本身、只有 Light/Dark 两格，照搬会错）。

### 未执行的验证

无。

### 风险分类

- [x] 无已知风险

### 影响与回滚

- 影响范围：仅治理文档；产品行为、构建、CI 均零变化
- 回滚 / 降级方式：revert 本 PR 即回到 #3609 合入后的状态
- 相邻改动提示：在办的 DS-2a 会改同一文件的 §12 PR 列（回填 DS-2a 自身编号），后合入的一方需一次琐碎 rebase

### 提交前检查

- [x] 已 review 完整 diff
- [x] 每个 commit 都带 DCO 签名（`git commit -s`，见 [DCO](../DCO)）
- [x] UI 改动已在「UI 变化」注明引用的设计规范章节（不涉及 UI 则跳过）
- [x] 未提交凭证、令牌或授权文件
- [x] 已补充必要文档
- [x] 已确认测试结果或说明未执行原因

### CODE REVIEW

**Comment by MagicLizi** [comment]
@kirozeng 👋 这个 PR 还有 8 条 review conversation 没 resolve(docs/design-rules/design-governance.md),auto-review 因此暂时跳过、没法继续审查 / 合并。

如果你已经按评论改完或回应了,请到对应 thread 上点 **Resolve conversation**;全部 resolve 后,下一轮 auto-review 会自动重新审查这个 PR。

**Comment by kirozeng** [comment]
已按 6 条 review 意见逐条修订并推送（`40990cd9e`），全部为文档收口、零视觉：

1. **保护值审批路径（P1）**：§1.1 新增「保护值」段——CINDY 皮肤族须用户裁决（`DESIGN.md §15` 实现期零裁量）、U2 二级信息色改值须新用户裁决（`§15.5` never darken unilaterally，冻结测试 `cindyThemes.test.ts` 组⑦把关）、`annotation-accent` 冻结不改（`§15.4` exempt, do not change）。记账行范围同步收窄为「非保护区观感」，不再笼统写成「设计师批即可改」。
2. **豁免路径（P2）**：禁止事项收窄为「**未经裁决**加豁免」；真实例外仍走 §11 的 `hardcoded-color-exemptions.json` 登记路径（附理由与 owner），与为消红灯私自加条目区分开。
3. **扫描能力描述（P2）**：§13.1 改为如实边界——`hardcoded-color-audit` 扫 diff 全部新增行、内联样式 `#hex`/`rgb()`/`hsl()` 字面量会被发现；真正扫不到的是正则表达不了的动态值与 canvas/xterm 自绘内容。
4. **DS-1 合入日期（P2）**：勘误为 2026-08-30（merge commit `2dc8cf123` 实际时间 00:20:05 +0800）。
5. **新代码语义约束（P2）**：§13.3 恢复强制表述——新代码**必须**走语义层（§1.1 管道规则 + `DESIGN.md §10` 禁止硬编码），仅存量渐进。
6. **台账唯一性范围（P2）**：§1.1 补「迁移台账」限定（只指 §2.1 的 `design-inventory.md`，`token-decision-table.md` / `design-decision-log.md` 等职责不同的台账不在此列）。

索引版本记录（`cindy-design-system.md`）已同步。验证：`hardcoded-color-audit` 对本 PR diff `unexpected=0`（本次新增行无色值字面量）、`check:i18n-glossary` 通过。

另说明 Windows unit tests 红灯：两个失败（`maker-pi-manager` edge-cases 5c 时序敏感偶发、`codexProxyHost` 15s 超时）均为本 PR 之外路径的既有 flaky——本 PR 全部改动只含 2 个 docs 文件，同一 SHA 区间 main 分支 Windows 全绿；未在本 PR 改测试（不在范围且属 CI 路径），建议对失败 job 重跑确认。

dispatch:eb7db94ff125d64b

**Comment by MagicLizi** [comment]
@kirozeng 👋 这个 PR 还有 9 条 review conversation 没 resolve(docs/design-rules/design-governance.md),auto-review 因此暂时跳过、没法继续审查 / 合并。

如果你已经按评论改完或回应了,请到对应 thread 上点 **Resolve conversation**;全部 resolve 后,下一轮 auto-review 会自动重新审查这个 PR。

**Comment by kirozeng** [comment]
已按新的 review 意见修订并推送（`ccdd98ce8`），零视觉纯文档：

**保留 §3.3/§3.4 既有准入（P2）**：核实成立——上一轮把 §13.3 改成「新代码**必须**走语义层」是绝对表述，与本文 §3.3（品牌表达、兼容合同或台账登记为 `protected` 的 surface 允许直接依赖 reference）、§3.4（运行时派生值留在代码中不强迁）冲突，会让后续 review 或 DS-7 棘轮把合同本身批准的入口当违规拦下。改为「新代码**默认**走语义层」，并显式点出 §3.3/§3.4 是合同开的准入、不是「优先语义层」原则的漏洞。

验证：`hardcoded-color-audit` 对本次新增行 `unexpected=0`、`check:i18n-glossary` 通过。

**关于 Windows unit tests「required context 缺席」**：核实当前是 `pending`（还在跑，不是真失败），本轮不做额外处理，等 CI 出结果。

dispatch:a2dc4d95f503719f

File: docs/design-rules/design-governance.md [hunk 1]
```diff
@@ -19,6 +19,20 @@ Cindy 的新界面大量由非设计师贡献。本合同要达成的状态：
...
+
+本合同的全部规则分两类，性质不同、改法不同：
+
+| | 是什么 | 能不能改 |
+| --- | --- | --- |
+| **管道**（永久规则） | 值只有一个出处；旧 Token ID 不删不改名；用户主题不改写磁盘；颜色走语义 token 不许硬编码；台账只有一份 | **不许绕过**——它与长相无关，改版后依然成立 |
+| **记账**（当前拍板值） | 具体色号、圆角三档的当前成员、保护区观感（CINDY 皮肤族、登录链路、语义豁免色） | **随时能改**——走下面的合法改值路径 |
+
+**守卫红灯不是禁令，是提醒走合法路径**：改数值真相源 → 同一 PR 更新对应守卫快照 /
+台账 / 文档 → 按 §6 交证据 → 设计师批准。这是必要步骤，不是绕守卫。唯一禁止的事：
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§1.1 现写明保护值不适用通用记账改值路径，CINDY 皮肤族实现期零裁量、U2 须新的用户裁决、annotation-accent 明确 do not change。原「设计师批准即可改保护值」的问题已不成立。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 2]
```diff
@@ -19,6 +19,20 @@ Cindy 的新界面大量由非设计师贡献。本合同要达成的状态：
...
+为了让红灯变绿而加豁免、绕加载路径、在别处重声明——那是砸管道，不是改设计。系统对
+「混乱」的全部定义：消灭**没人决定过的变化**；一切拍板过的值都可以再拍板。
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§1.1 现将禁止事项限定为「为了让红灯变绿而未经裁决加豁免」，并写明真实例外仍走 §11 hardcoded-color-exemptions.json 经裁决登记。正式豁免路径未再被绝对禁止。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 3]
```diff
@@ -235,3 +250,15 @@ ls apps/desktop/src/renderer/themes/builtin/*.ts | wc -l
 
 本节随施工推进更新完成状态；十张全部合入后，本节归档进
 [`design-decision-log.md`](./design-decision-log.md)，本文其余章节转入长期生效。
+
+## 13. 已知边界（如实登记，不夸大机器能力）
+
+以下是本治理体系**做不到**的事。登记它们的目的：不得把「机器没拦」当作「合规」的证据。
+
+1. **棘轮管不到内联样式与非 CSS 绘制**——`style={{...}}` 的 JS 字面量、canvas / xterm
+   一类自绘内容不经过样式文件，机器扫不到；靠 review 与 §11 登记的
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§13.1 现写明 hardcoded-color-audit 扫 diff 全部新增行，TSX 内联 style 的 hex/rgb/hsl 会被发现；真正扫不到的是动态拼接与 canvas/xterm 自绘。原「机器扫不到内联颜色」的绝对表述已删除。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 4]
```diff
@@ -222,7 +237,7 @@ ls apps/desktop/src/renderer/themes/builtin/*.ts | wc -l
 
 | # | 标题 | 风险类别 | PR |
 | --- | --- | --- | --- |
-| DS-1 | `docs(design-system): DS-1 建立治理合同与存量门禁处置表`（本张） | 零视觉（纯文档） | — |
+| DS-1 | `docs(design-system): DS-1 建立治理合同与存量门禁处置表` | 零视觉（纯文档） | ✅ #3609（2026-08-29 合入） |
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§12 路线图表 DS-1 现记为 ✅ #3609（2026-08-30 合入），与 merge commit 日期一致。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 5]
```diff
@@ -235,3 +250,15 @@ ls apps/desktop/src/renderer/themes/builtin/*.ts | wc -l
...
+
+以下是本治理体系**做不到**的事。登记它们的目的：不得把「机器没拦」当作「合规」的证据。
+
+1. **棘轮管不到内联样式与非 CSS 绘制**——`style={{...}}` 的 JS 字面量、canvas / xterm
+   一类自绘内容不经过样式文件，机器扫不到；靠 review 与 §11 登记的
+   `check-pr-design-basis` 必读要求约束；
+2. **「复用道路唯一」是纪律不是机器闸**——防止出现第二套 Button / Toast / 菜单动作模型，
+   靠 review 裁决，机器只能发现雷同、不能自动判定谁该让位；
+3. **语义分层是渐进的**——存量代码混引语义层与原始值是登记在案的现状；新代码优先语义层，
+   存量随各表面迁移顺带收敛，不专门开重构 PR。
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§13.3 现写「新代码默认走语义层」，并回指 §1.1 管道规则与 DESIGN.md §10 禁止新代码硬编码颜色/裸设计值；§3.3 准入被明确为合同开的口子而非把强制降成偏好。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 6]
```diff
@@ -19,6 +19,20 @@ Cindy 的新界面大量由非设计师贡献。本合同要达成的状态：
 范围：Desktop 与 Mobile 的 UI 视觉基础层（颜色、排版、间距、圆角、组件样式）。产品信息
 架构与交互流程的设计不在本合同范围内。
 
+### 1.1 管道与记账（守卫红灯的正确读法）
+
+本合同的全部规则分两类，性质不同、改法不同：
+
+| | 是什么 | 能不能改 |
+| --- | --- | --- |
+| **管道**（永久规则） | 值只有一个出处；旧 Token ID 不删不改名；用户主题不改写磁盘；颜色走语义 token 不许硬编码；台账只有一份 | **不许绕过**——它与长相无关，改版后依然成立 |
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§1.1 管道行现写「迁移台账只有一份（见 §2.1；token-decision-table.md、design-decision-log.md 等职责不同的台账不在此列）」，范围已限定。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§1.1 禁止的是未经裁决为消红灯加豁免；资产固有色/平台语义色等真实例外仍走 §11 豁免表并附理由与 owner。与 DESIGN.md 登记路径一致。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 7]
```diff
@@ -235,3 +250,15 @@ ls apps/desktop/src/renderer/themes/builtin/*.ts | wc -l
...
 本节随施工推进更新完成状态；十张全部合入后，本节归档进
 [`design-decision-log.md`](./design-decision-log.md)，本文其余章节转入长期生效。
+
+## 13. 已知边界（如实登记，不夸大机器能力）
+
+以下是本治理体系**做不到**的事。登记它们的目的：不得把「机器没拦」当作「合规」的证据。
+
+1. **棘轮管不到内联样式与非 CSS 绘制**——`style={{...}}` 的 JS 字面量、canvas / xterm
+   一类自绘内容不经过样式文件，机器扫不到；靠 review 与 §11 登记的
+   `check-pr-design-basis` 必读要求约束；
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§13.1 已区分「内联源码字面量会被 audit 扫到」与「动态值/canvas/xterm 才扫不到」。原覆盖面失实已纠正。代为 resolve；有异议可 reopen。

File: docs/design-rules/design-governance.md [hunk 8]
```diff
@@ -235,3 +264,18 @@ ls apps/desktop/src/renderer/themes/builtin/*.ts | wc -l
...
+
+1. **正则扫描的边界**——`hardcoded-color-audit` 扫 diff 全部新增行，`style={{...}}`
+   TSX 内联样式里的 `#hex` / `rgb()` / `hsl()` 字面量**会被发现**（它不按文件类型过滤）。
+   真正扫不到的是：正则表达不了的动态值（变量拼接、数值通道、运行期计算结果）与
+   canvas / xterm 一类自绘内容——这些不经过样式文件，靠 review 与 §11 登记的
+   `check-pr-design-basis` 必读要求约束；
+2. **「复用道路唯一」是纪律不是机器闸**——防止出现第二套 Button / Toast / 菜单动作模型，
+   靠 review 裁决，机器只能发现雷同、不能自动判定谁该让位；
+3. **语义分层是渐进的**——存量代码混引语义层与原始值是登记在案的现状；**新代码必须走
+   语义层**（§1.1 管道规则 + `DESIGN.md §10` 均禁止新代码硬编码颜色、引入裸设计值），
```
[the remark this answers is not in the document]
Comment (MagicLizi) [reply docs/design-rules/design-governance.md]: 已在 ccdd98ce 核实：§13.3 与 §3.3 现保留品牌表达/兼容合同/台账 protected surface 直接依赖 reference 的准入，以及 §3.4 运行期派生值，并写明这是合同本身开的口子。代为 resolve；有异议可 reopen。

**Review by MagicLizi — APPROVED** [review]
独立审查通过：纯文档修订治理合同（管道/记账、保护值门槛、豁免路径、audit 覆盖面、DS-1 回填），未削弱管道红线或 CI 管理审核门。未发现 P0/P1。

**Comment by MagicLizi** [comment]
把管道和记账拆开、把保护值门槛写进合同之后，后面的 DS 施工终于能改数值而不必绕守卫。已知边界那一节也避免了把「机器没拦」当成合规。
